### PR TITLE
Spessgen Templates

### PR DESCRIPTION
**Pull request by Pyko1**
Broken Mimeship:
mimes

### CODE REVIEW

**Comment by Pyko1** [comment]
Original Clown Ship
originalcloonship

**Comment by Pyko1** [comment]
Temple of Unholy Geometer

The blood track around is actually invoked shield runes, and on every pentagram there is faithless mob.
Notable loot: 4 arcane tomes (useless for non cultists) and Veil render in the middle
capture

**Comment by Carbonhell** [comment]
Veil render spawns narsie...change the loot into something less destructive

**Comment by Pyko1** [comment]
3 more asteroids and I assume it would be great to have a lot more of these "non-meaningful templates" for space generation so ships and such get a bit rarer.

5x5
6x10
16x17

**Comment by Pyko1** [comment]
Changed it to lesser Veil Render, which spawns 8 hostile faithless.

**Comment by Pyko1** [comment]
Pod crash:
Loot: Basic engineer gear.

capture

**Comment by Pyko1** [comment]
Created Runed_Metal Mineral door.
As well as set the default icon_state for every mineral_door to fix the issue of all of them showing as metal doors when mapping.

Used the said runed_metal mineral door on the temple template entrances.

runed_metal_door

**Comment by Nokington** [comment]
:+1:

**Comment by Pyko1** [comment]
Updated the temple picture to reflect changes on it.

capture

**Comment by Memendia** [comment]
@Pyko1 

All hail the one true god!
*i fucking love that unholy temple*

**Comment by Pyko1** [comment]
I'm pretty happy with it as well, only issue I really have is the bad icon on pylons. The Icon really should be 96x96, and appear 1,5-2 tiles tall and have some sort of perspective.

**Comment by danhibiki** [comment]
Looks pretty cool so far, I assume your adding more? So far :+1:

**Comment by doobiedoo23** [comment]
This is great

**Comment by Pyko1** [comment]
Syndicate Scout Ship
Notable loot: Retro laser, syndicate toolbox, fireaxe, and the loot from spaceproof ranged/melee syndicate mobs, if you manage to kill em.
The computers are Teleporter, 2x basic computers (does nothing), cameras, crewmonitor, and medical records.

capture

**Comment by Pyko1** [comment]
Abandoned Abductor Ship
The walls are breakable, generally has everything that abductor ships have, except crew, and abductor console.
Notable loot alien pistol with generic pin, and abductor stun baton, as well as abductor surgical gear, and handcuffs.
The X is skeleton.

capture

**Comment by Nebyab** [comment]
aure

**Comment by TDcoolguy300** [comment]
I am a tad uneasy on having that nuclear warhead on the Clown ship.

**Comment by Pyko1** [comment]
Don't be such a killjoy, I bet everyone is going to have a blast when their prank unfolds.

**Comment by doobiedoo23** [comment]
honk

**Comment by Pyko1** [comment]
Even tho, I think I'll rename it to Thermo Honkular Device, and make it accept bikehorn as a disc.
And set the code to HONK, only a clown can somehow manage to enter that with a numpad.

**Comment by TDcoolguy300** [comment]
Perfect

**Comment by Kokojos** [comment]
Absolutely! :+1:

**Comment by Pyko1** [comment]
Eww, gotta check all the templates for roundstart AT's as well, at least the mimeship has them, and some of the existing ones as well.

**Comment by Pyko1** [comment]
Turned everything in z3 in to templates located in templates/z3. Maybe do same for z4, and then empty both z3 and z4 and use them also for templates spawning, and move the content in spacegen and up the amount of spawned templates?

**Comment by Pyko1** [comment]
Added z4 templates as well to its own location, So everything outside of station, mining and CentComm could be technically randomized.

File: code/game/gamemodes/wizard/artefact.dm [hunk 1]
```diff
@@ -72,6 +72,14 @@
 	activate_descriptor = "hunger"
 	rend_desc = "Reverberates with the sound of ten thousand moos."
 
+/obj/item/weapon/veilrender/lesserveilrender
+	name = "lesser veil render"
+	desc = "A wicked curved blade of alien origin, recovered from the ruins of ancient temple."
```
Comment (TDcoolguy300) [inline code/game/gamemodes/wizard/artefact.dm]: "recovered from the ruins of ancient temple." there should be an "an" between "of" and "ancient"

**Comment by nalarmg** [comment]
Pending code review comment addressing before merge.

**Comment by JustinBNewton** [comment]
Code for new walls and objects looks good and functions as intended.

**Pull request merged**